test/hotspot/gtest/aarch64/asmtest.out.h

```diff
@@ -540,131 +540,135 @@
     __ sminv(v19, __ T8H, v20);                        //       sminv   h19, v20.8H
```

Comment:
A manual merge seems not working fine for this file to resolve conflicts. I will sent you the regenerated file.

Reply:
@nsjian Thanks a lot. I pushed the regenerated asmtest.out.h.